src/api.js

```diff
@@ -53,6 +53,7 @@ export const getClient = async (UIStore) => {
   const dedupVehiclesLink = new ApolloLink((operation, forward) => {
     return forward(operation).map((data) => {
       if (data.data.vehicles.length > 5000) {
+        console.log("boop");
```

Comment:
o.O

Reply:
That's a very important part of the feature.